src/Form/Procuration/AbstractProcurationType.php

```diff
@@ -64,14 +61,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'widget' => PhoneNumberType::WIDGET_COUNTRY_CHOICE,
             ])
             ->add('emailAddress', EmailType::class)
-            ->add('birthdate', BirthdayType::class, [
-                'widget' => 'choice',
-                'years' => $options['years'],
-                'placeholder' => [
-                    'year' => 'AAAA',
-                    'month' => 'MM',
-                    'day' => 'JJ',
-                ],
+            ->add('birthdate', DatePickerType::class, [
+                'max_date' => new \DateTime('-17 years'),
```

Comment:
do we need to use min_date too?

Reply:
No, entity validation does not require min date.

Reply:
yes, but before, the `years` select had the min value = NOW - 120 = min_date for the new formType, no?